src/meta/processors/jobMan/MetaJobExecutor.cpp

```diff
@@ -138,12 +146,67 @@ ErrOrHosts MetaJobExecutor::getLeaderHost(GraphSpaceID space) {
     return hosts;
 }
 
+ErrOrHosts MetaJobExecutor::getListenerHost(GraphSpaceID space) {
+    // TODO : Only ES listener is supported
```

Comment:
Better handle it in this PR, support a interface of `getListenerHost(space, type)`, you could use the `paras_`

Reply:
Good idea.